main.go

```diff
@@ -24,8 +24,8 @@ import (
 	volrep "github.com/csi-addons/volume-replication-operator/api/v1alpha1"
 	spokeClusterV1 "github.com/open-cluster-management/api/cluster/v1"
 	ocmworkv1 "github.com/open-cluster-management/api/work/v1"
```

Comment:
The api repository [go.mod](https://github.com/stolostron/api/blob/a6845f2ebcb186ec26b832f60c988537a58f3859/go.mod#L1) has not changed its name. For example as compared to say placementRule [go.mod](https://github.com/stolostron/multicloud-operators-placementrule/blob/8eedb3fa9ff6b619829e4cc3c099ef594e96f3cf/go.mod#L1)

Reply:
I presume you ran `go mod tidy`, but maybe something else is needed to purge the go module "cache"...

Reply:
I experimented with it and I think that not all references were fixed....and this is one that hasn't been...
```sh
$ make
main.go:25:2: no required module provides package github.com/stolostron/api/work/v1; to add it:
        go get github.com/stolostron/api/work/v1

$ go get github.com/stolostron/api/work/v1
go: downloading github.com/stolostron/api v0.0.0-20210527013639-a6845f2ebcb1
go get: github.com/stolostron/api@v0.0.0-20210527013639-a6845f2ebcb1: parsing go.mod:
        module declares its path as: github.com/open-cluster-management/api
                but was required as: github.com/stolostron/api

$ go mod tidy
go: finding module for package github.com/stolostron/api/work/v1
go: found github.com/stolostron/api/work/v1 in github.com/stolostron/api v0.0.0-20210527013639-a6845f2ebcb1
go: github.com/ramendr/ramen imports
        github.com/stolostron/api/work/v1: github.com/stolostron/api@v0.0.0-20210527013639-a6845f2ebcb1: parsing go.mod:
        module declares its path as: github.com/open-cluster-management/api
                but was required as: github.com/stolostron/api
```
I presume [this line](https://github.com/stolostron/api/blob/a6845f2ebcb186ec26b832f60c988537a58f3859/go.mod#L1) is the problem:

```golang
module github.com/open-cluster-management/api
```

so no fix needed here.  Instead of opening a stolostron issue I suggest this be fixed by moving to the upstream `open-cluster-management-io` version.  Issue #385 tracks this.

Reply:
Ack!